internal/system/impl/sqlstore.go

```diff
@@ -30,28 +30,49 @@ const (
 
 	// DefaultMetadataImage is the default image for table's metadata.
 	DefaultMetadataImage = "https://bafkreifhuhrjhzbj4onqgbrmhpysk2mop2jimvdvfut6taiyzt2yqzt43a.ipfs.dweb.link"
+
+	// DefaultAnimationURL is an empty string. It means that the attribute will not appear in the JSON metadata.
+	DefaultAnimationURL = ""
 )
 
 // SystemSQLStoreService implements the SystemService interface using SQLStore.
 type SystemSQLStoreService struct {
-	extURLPrefix        string
-	metadataRendererURI string
-	stores              map[tableland.ChainID]sqlstore.SystemStore
+	extURLPrefix         string
+	metadataRendererURI  string
+	animationRendererURI string
+	stores               map[tableland.ChainID]sqlstore.SystemStore
 }
 
 // NewSystemSQLStoreService creates a new SystemSQLStoreService.
 func NewSystemSQLStoreService(
 	stores map[tableland.ChainID]sqlstore.SystemStore,
 	extURLPrefix string,
 	metadataRendererURI string,
+	animationRendererURI string,
 ) (system.SystemService, error) {
 	if _, err := url.ParseRequestURI(extURLPrefix); err != nil {
 		return nil, fmt.Errorf("invalid external url prefix: %s", err)
 	}
+
```

Comment:
@jsign moved both uris validations to here

Reply:
SGTM